indexer/borders.cpp

```diff
@@ -0,0 +1,95 @@
+#include "indexer/borders.hpp"
+
+#include "coding/file_container.hpp"
+#include "coding/geometry_coding.hpp"
+#include "coding/var_record_reader.hpp"
+
+#include "base/logging.hpp"
+#include "base/macros.hpp"
+
+#include "defines.hpp"
+
+using namespace std;
+
+namespace
+{
+template <class ReaderT>
+class BordersVector
+{
+  DISALLOW_COPY(BordersVector);
```

Comment:
This is usually the last line, not the first.

Reply:
done